features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/blocks/api-availability/ajax/api-availability.jag

```diff
@@ -66,10 +66,35 @@ include("/jagg/jagg.jag");
 
 
             dataResult = mod.getDataFromTable(tableName, fullSearchQuery, startindex, count, "api_version", sortOrderBool);
-            countResult = mod.getCountFromTable(tableName, fullSearchQuery);
 
             parsedDataResult = JSON.parse(dataResult);
 
+            var temp1, temp2;
+            var apiDetails, apiProvider, apiName, apiVersion, apiId;
+            var isExists;
+            var modifiedDataResult= new Array();
+
+            var APIManagerFactory = Packages.org.wso2.carbon.apimgt.impl.APIManagerFactory;
+            var APIIdentifier = Packages.org.wso2.carbon.apimgt.api.model.APIIdentifier;
+
+            for (i = 0; i < parsedDataResult.length; i++) {
+                apiDetails= parsedDataResult[i]["values"]["api_version"];
+                temp1 = apiDetails.split("--");
+                apiProvider = temp1[0];
+                temp2 = temp1[1].split(":");
+                apiName = temp2[0];
+                apiVersion = temp2[1].substring(1);
+                apiId = apiProvider + "_" + apiName + "_" + apiVersion;
+
+                isExists = APIManagerFactory.getInstance().getAPIProvider(provider).isAPIAvailable(new APIIdentifier(apiId));
```

Comment:
Please use the other constructor which creates an APIIdentifier using provider, name and version. If an API name/version contains an "_" this will break.